src/features/apiDelegation/offered/ChooseApiPage/ChooseApiPage.tsx

```diff
@@ -167,17 +163,18 @@ export const ChooseApiPage = () => {
     );
   });
 
-  const chosenDelegableOrgs = chosenOrgs.map((org: DelegableOrg) => {
+  const skeleton = () => {
+    const skeletonHeight = '66px';
     return (
-      <CompactDeletableListItem
-        key={org.orgNr}
-        startIcon={<Buldings3Icon />}
-        removeCallback={chosenOrgs.length > 1 ? () => dispatch(softRemoveOrg(org)) : null}
-        leftText={org.orgName}
-        middleText={!isSm ? t('api_delegation.org_nr') + ' ' + org.orgNr : undefined}
-      ></CompactDeletableListItem>
+      <>
+        <Skeleton.Rectangle height={skeletonHeight}></Skeleton.Rectangle>
+        <Skeleton.Rectangle height={skeletonHeight}></Skeleton.Rectangle>
+        <Skeleton.Rectangle height={skeletonHeight}></Skeleton.Rectangle>
+        <Skeleton.Rectangle height={skeletonHeight}></Skeleton.Rectangle>
+        <Skeleton.Rectangle height={skeletonHeight}></Skeleton.Rectangle>
```

Comment:
I suggest removing one of these rectangles, since 5 large blocks on both sides feels a bit heavy

Reply:
I don't discuss or apply code using feelings as guideline and if we were to discuss feelings all the time but if I were to use feelings as an argument I could have just said that I felt 5 large blocks was just about right as an counterargument and the discussion would stall.

However the reason why there are 5 large blocks there is because when the view is done loading it is displayed 5 ActionBars there. However I can agree on having these 5 rectangles on both sides for "delegerbare api" and "valgte API" is unneccessary and we could reduce to having them just on the delegerbare api-side or just have 5 action bars spanning over the whole container.

Reply:
I'm not talking about feeling as in emotions here, I'm talking about the impression it has on the user.
10 large blocks of grey, with minimal padding between them, is very heavy visually, pulling focus and taking up space like a brick wall in the middle of the page - to the user, it will _feel heavy_.
This is especially true when most of this will be replaced with blankness a second later to it can end up feeling like a hit-and-run meeting with a truck.

Part of this can be helped by removed by using fewer rectangles, but it would be even to, as you say, removing the skeletons on the right (I just didn't think I'd get you to do that) 😅
Since there sometimes can be some pre-chosen actionBars, we could also consider having just one or two on the right side. Idk, I'd have to see the different options to see what works the best.